packages/service-library/tests/long_running_tasks/test_long_running_tasks_task.py

```diff
@@ -29,12 +30,17 @@
 from tenacity.stop import stop_after_delay
 from tenacity.wait import wait_fixed
 
-_RETRY_PARAMS = dict(
-    reraise=True,
-    wait=wait_fixed(0.1),
-    stop=stop_after_delay(60),
-    retry=retry_if_exception_type(AssertionError),
-)
+_RETRY_PARAMS = {
+    "reraise": True,
+    "wait": wait_fixed(0.1),
+    "stop": stop_after_delay(60),
+    "retry": retry_if_exception_type(
+        (
+            BaseException,  # retry when pytest reports `Failed: DID NOT RAISE``
```

Comment:
really? you mean that with pytest.raises does not raise an AssertionError when there is no exception???

Reply:
yeah, but this is a bad pattern, so I will avoid catching this one.